articles/connections/social/apple.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Add Sign in with Apple to Your App
+connection: Apple
+index: 3
+image: /media/connections/apple.svg
+seo_alias: apple
+description: Learn how to add login functionality to your app with Apple. You will need to generate keys, copy these into your Auth0 settings, and enable the connection.
+toc: true
+topics:
+  - authentication
+  - connections
+  - social
+  - apple
+contentType: how-to
+useCase:
+  - add-login
+  - customize-connections
+  - add-idp
+---
+# Add Sign in with Apple to Your App
+
+This guide will show you how to add functionality to your web app that allows your users to use Apple's "Sign in with Apple" functionality. Using the Apple connection will require the following:
```

Comment:
Let's mention it's not required to configure it this way, and you can use it with Auth0's developer keys for testing/dev purposes.